src/checkpoints/checkpoints.service.ts

```diff
@@ -83,17 +83,21 @@ export class CheckpointsService {
     ticker: string,
     signerDto: TransactionBaseDto
   ): ServiceReturn<Checkpoint> {
-    const { signer, webhookUrl } = signerDto;
+    const { signer, webhookUrl, dryRun } = signerDto;
     const asset = await this.assetsService.findOne(ticker);
 
-    return this.transactionsService.submit(asset.checkpoints.create, {}, { signer, webhookUrl });
+    return this.transactionsService.submit(
+      asset.checkpoints.create,
+      {},
+      { signer, webhookUrl, dryRun }
+    );
   }
 
   public async createScheduleByTicker(
     ticker: string,
     createCheckpointScheduleDto: CreateCheckpointScheduleDto
   ): ServiceReturn<CheckpointSchedule> {
-    const { signer, webhookUrl, ...rest } = createCheckpointScheduleDto;
+    const { signer, webhookUrl, dryRun, ...rest } = createCheckpointScheduleDto;
```

Comment:
With the suggested helper it will be hard to miss cases like this.

If this is going to be left then it needs to be passed in.